src/sdgio-edit.owl

```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020244> <https://clarisa.cgiar.org/api/srf-idos/7>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020244> "improved human and animal health through better agricultural practices"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020244>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020245> (natural capital enhanced and protected, especially from climate change)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020245> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020245> "2019-04-29T16:29:07Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020245> <https://clarisa.cgiar.org/api/srf-idos/8>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020245> "natural capital enhanced and protected, especially from climate change"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020245>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020246> (enhanced benefits from ecosystem goods and services)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020246> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020246> "2019-04-29T16:29:41Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020246> <https://clarisa.cgiar.org/api/srf-idos/9>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020246> "enhanced benefits from ecosystem goods and services"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020246>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020247> (more sustainably managed agro-ecosystems)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020247> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020247> "2019-04-29T16:30:19Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020247> <https://clarisa.cgiar.org/api/srf-idos/10>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020247> "more sustainably managed agro-ecosystems"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020247>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020248> (mitigation and adaptation achieved)
```

Comment:
We'll need to ask for what the CGIAR criteria for the achievement of this and similarly worded objective are.

Reply:
I added issue #150.